Error Type: System.InvalidOperationException
Error Detail: Unable to setup default configuration for 'MFDLabs.Grid.Bot.Properties.Settings' for guild '883742755287875655': There was no default configuration at the path 'client-products/discord-guild-settings/Debug/MFDLabs.Grid.Bot.Properties.Settings/default'
Inner Exception: 
Exception Stack Trace: 
   at MFDLabs.Discord.Configuration.VaultConfigurationClient.SetupDefaultConfiguration(SocketMessage message, String groupName) in C:\git\MFDLabs\MFDLabs.Grid\Assemblies\MFDLabs.Discord.Configuration\Clients\VaultConfigurationClient.cs:line 122
   at MFDLabs.Discord.Configuration.VaultConfigurationClient.GetSettingsWithMetaValuesNoCache(SocketMessage message, String groupName) in C:\git\MFDLabs\MFDLabs.Grid\Assemblies\MFDLabs.Discord.Configuration\Clients\VaultConfigurationClient.cs:line 160
   at MFDLabs.Discord.Configuration.VaultConfigurationClient.GetMetaValue(SocketMessage message, String groupName, String settingName) in C:\git\MFDLabs\MFDLabs.Grid\Assemblies\MFDLabs.Discord.Configuration\Clients\VaultConfigurationClient.cs:line 217
   at MFDLabs.Discord.Configuration.DiscordConfigurationHelper.CheckCanRead(SocketMessage message, String groupName) in C:\git\MFDLabs\MFDLabs.Grid\Assemblies\MFDLabs.Discord.Configuration\DiscordConfigurationHelper.cs:line 134
   at MFDLabs.Discord.Configuration.DiscordConfigurationHelper.GetSetting[T](SocketMessage message, String groupName, String settingName) in C:\git\MFDLabs\MFDLabs.Grid\Assemblies\MFDLabs.Discord.Configuration\DiscordConfigurationHelper.cs:line 67
   at MFDLabs.Grid.Bot.Properties.SettingsProvider.GetSetting[T](SocketMessage message, String settingName) in C:\git\MFDLabs\MFDLabs.Grid\MFDLabs.Grid.Bot\Properties\SettingsProvider.cs:line 23
   at MFDLabs.Grid.Bot.Events.OnMessage.<Invoke>d__0.MoveNext() in C:\git\MFDLabs\MFDLabs.Grid\MFDLabs.Grid.Bot\Events\OnMessage.cs:line 22
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable.ConfiguredTaskAwaiter.GetResult()
   at Discord.EventExtensions.<InvokeAsync>d__1`1.MoveNext() in C:\git\MFDLabs\MFDLabs.Grid\Dependencies\Discord.Net.Core\Utils\AsyncEvent.cs:line 48
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable.ConfiguredTaskAwaiter.GetResult()
   at Discord.WebSocket.DiscordSocketClient.<TimeoutWrap>d__160.MoveNext() in C:\git\MFDLabs\MFDLabs.Grid\Dependencies\Discord.Net.WebSocket\DiscordSocketClient.cs:line 2983
Exception Source: MFDLabs.Discord.Configuration
Exception TargetSite: Void SetupDefaultConfiguration(Discord.WebSocket.SocketMessage, System.String)
Exception Data: System.Collections.ListDictionaryInternal
